msteams-platform/TOC.yml

```diff
@@ -176,7 +176,7 @@
         href: resources/bot-v3/bots-notifications.md
       - name: Context for bots
         href: resources/bot-v3/bots-context.md
-      - name: Send and receiving files
```

Comment:
Let's remove Gerunds from the TOC altogether. @surbhigupta12 please assign a dedicated task to someone to do this.

Specifically, such usage as `send and receiving` is not a good TOC entry.